pkgs/development/python-modules/advantage_air/default.nix

```diff
@@ -0,0 +1,26 @@
+{ aiohttp, buildPythonPackage, fetchPypi, lib, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "advantage_air";
+  version = "0.2.1";
```

Comment:
The latest release is [0.2.2](https://github.com/Bre77/advantage_air/commit/f7963d16ea782e84595b55663971d0f303586dec).

Reply:
Haha, I wasn't expecting the dev to get back to my issue so quickly. See Bre77/advantage_air#1 for context.

Reply:
Yeah, 0.2.2 was shortly released after this PR was made.